src/app/components/organisms/Tabs/Tabs.tsx

```diff
@@ -5,6 +5,7 @@ import { isEmpty } from '../../../utils/functions/validateFunctions';
 export interface Tab {
     content: ReactNode;
     isDisabled?: boolean;
+    padding?: { bottom: number; left: number; right: number; top: number };
```

Comment:
I don't get why you want to have all the paddings when you only need a top correction